src/test/java/seedu/address/logic/parser/TutorsPetParserTest.java

```diff
@@ -145,6 +146,12 @@ public void parseCommand_listModuleClass() throws Exception {
                 instanceof ListModuleClassCommand);
     }
 
+    @Test
+    public void parseCommand_list() throws Exception {
+        assertTrue(parser.parseCommand(ListCommand.COMMAND_WORD) instanceof ListCommand);
+        assertTrue(parser.parseCommand(ListCommand.COMMAND_WORD + " 3") instanceof ListCommand);
```

Comment:
"3" seems like a magic number across the test script. We might want to consider abstracting it into a variable in a separate PR

Reply:
agree with this